### ISSUE TITLE
Ps3NetServer does not mount folder

### ISSUE THREAD

**Issue by Immortaloren** [d-two/hassio-addons#6]
Can't start ps3netserver.
Launch log:
"Add-on: Ps3NetServer
 This is a Docker container for ps3netsrv (or ps3netsvr).
 Add-on version: 0.1.3-NAS
 You are running the latest version of this add-on.
 System: Debian GNU/Linux 11 (bullseye)  (aarch64 / qemuarm-64)
 Home Assistant Core: 2024.9.3
 Home Assistant Supervisor: 2024.09.1
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
s6-rc: info: service base-addon-banner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service base-addon-log-level: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service base-addon-log-level successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/00_smb
Mounting smb share(s)...
[20:06:46] FATAL: Error, unable to mount 192.168.1.55/ps3 to /mnt/ps3 with username ps3netsrv, ps3netsrv. Please check your remote share path, username, password, domain, try putting 0 in UID and GID
[20:06:46] FATAL: Here is some debugging info :"

The Samba server with login and password is running, you can see it even from under Windows.
What am I doing wrong? There are no instructions on what to do or how to run it for lamers :(

**Comment by d-two** [comment]
> Can't start ps3netserver. Launch log: "Add-on: Ps3NetServer This is a Docker container for ps3netsrv (or ps3netsvr). Add-on version: 0.1.3-NAS You are running the latest version of this add-on. System: Debian GNU/Linux 11 (bullseye) (aarch64 / qemuarm-64) Home Assistant Core: 2024.9.3 Home Assistant Supervisor: 2024.09.1 Please, share the above information when looking for help or support in, e.g., GitHub, forums or the Discord chat. s6-rc: info: service base-addon-banner successfully started s6-rc: info: service fix-attrs: starting s6-rc: info: service base-addon-log-level: starting s6-rc: info: service fix-attrs successfully started s6-rc: info: service base-addon-log-level successfully started s6-rc: info: service legacy-cont-init: starting cont-init: info: running /etc/cont-init.d/00_smb Mounting smb share(s)... [20:06:46] FATAL: Error, unable to mount 192.168.1.55/ps3 to /mnt/ps3 with username ps3netsrv, ps3netsrv. Please check your remote share path, username, password, domain, try putting 0 in UID and GID [20:06:46] FATAL: Here is some debugging info :"
> 
> The Samba server with login and password is running, you can see it even from under Windows. What am I doing wrong? There are no instructions on what to do or how to run it for lamers :(

Please check the Readme... and compare it with your config...
I think you forget the '//'

**Comment by Immortaloren** [comment]
> Пожалуйста, проверьте файл Readme... и сравните его с вашей конфигурацией... Я думаю, вы забыли '//'


No, I haven't forgotten. Because if you don’t put “//” then it immediately writes - the format is not correct. I tried it in different ways. In the end, I came to the conclusion that the add-on does not work, because if I use https://github.com/shawly/docker-ps3netsrv with the same settings with the same folder, then everything works right away. Just enter the command 
docker run -d \
    --name=ps3netsrv \
    -p 38008:38008 \
    -v $HOME/ps3games:/games:rw \
    shawly/ps3netsrv

**Comment by d-two** [comment]
> > Пожалуйста, проверьте файл Readme... и сравните его с вашей конфигурацией... Я думаю, вы забыли '//'
> 
> 
> No, I haven't forgotten. Because if you don’t put “//” then it immediately writes - the format is not correct. I tried it in different ways. In the end, I came to the conclusion that the add-on does not work, because if I use https://github.com/shawly/docker-ps3netsrv with the same settings with the same folder, then everything works right away. Just enter the command 
> docker run -d \
>     --name=ps3netsrv \
>     -p 38008:38008 \
>     -v $HOME/ps3games:/games:rw \
>     shawly/ps3netsrv
> 

I use the addon without problems... 

```
Mounting smb share(s)...
[17:05:52] INFO: ... //192.168.137.100/home_files/Media/Share successfully mounted to /mnt/Share with options ,uid=0,gid=0,iocharset=utf8
cont-init: info: /etc/cont-init.d/00_smb exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service init-ps3netsrv: starting
```
